docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+
+version: "3.3"
+
+services:
+  mongodb:
+    ports:
+      - "27017:27017"
+    container_name: altv-mongo
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=root
```

Comment:
Shouldn't be hard coding these into the environment. It's a security risk and the secrets get baked into the image layers.

See: https://earthly.dev/blog/docker-secrets/

Reply:
no that's not hard. But as i want to mention; i need to refine this pr i think. I do this in 10 mins or so. Let me deep dive at this weekend.